Source/WTF/wtf/OptionSet.h

```diff
@@ -75,7 +75,7 @@ constexpr std::enable_if_t<std::is_enum_v<E>, bool> isValidOptionSetEnum(E e)
         return OptionSetValueChecker<std::underlying_type_t<E>, typename EnumTraits<E>::values>::isValidOptionSetEnum(static_cast<std::underlying_type_t<E>>(e));
```

Comment:
This code currently just checks that the value passed is equal to one of the values inside the enum (option set), but doesn't enforce that exactly one bit is set, which is why the `else` part trips over an assert.

We could keep it `hasOneBitSet`, but that does mean we'd have to rethink what default values to pass wherever those enums are used.